src/components/Header/Header.tsx

```diff
@@ -11,6 +11,11 @@ import Identity from './HeaderIdentity';
 import Navigation from './HeaderNavigation';
 
 interface HeaderProps {
+    /**
+     * The URL that the link in the upper left of the header will point to.
+     * @see https://rivet.uits.iu.edu/components/navigation/header/
+     */
+    headerUrl: string;
```

Comment:
My two cents is that I'd expect this to be named `href` to fall in line with the other components which have similar properties (button, etc).

Reply:
Agreed. Fixing.